### ISSUE TITLE
VideoStreamRendererView does not show anything , although VideoStreamRenderer is not null

### ISSUE THREAD

**Issue by bogdanmunteanu** [Azure/Communication#389]
**Describe the bug**
We have the remoteParticipant and remoteVideoStream not null , received from the Azure SDK , like this :
```
        val remoteParticipant = call?.remoteParticipants?.firstOrNull()
        val remoteVideoStream = remoteParticipant?.videoStreams?.firstOrNull()
```
and then , we are creating the VideoStreamRenderer like this :
`       remoteVideoStreamRenderer = VideoStreamRenderer(remoteVideoStream, context)
        remoteVideoStreamRendererView = remoteVideoStreamRenderer?.createView(CreateViewOptions(ScalingMode.FIT))
        binding?.llFullVideoContainer?.addView(remoteVideoStreamRendererView, 0)
`
After that , when the conference starts , the audio is on ( and we can hear the audio) but sometimes there is not video feed. 

**Expected behavior**
Video feed can be seen by both parties in the conference.

**Additional context**
Reproduction rate 50% (5/10 cases )

**Logs**
There are no logs available as the SDK does not throw any errors and the app does not crash.

**Comment by LoadLibrary** [comment]
@bogdanmunteanu , is the app listening to "addOnVideoStreamsUpdatedListener" ?

**Comment by LoadLibrary** [comment]
BTW, please find the instructions to source the logs at https://docs.microsoft.com/en-us/azure/communication-services/concepts/troubleshooting-info?tabs=csharp%2Candroid%2Cdotnet .

**Comment by bogdanmunteanu** [comment]
> @bogdanmunteanu , is the app listening to "addOnVideoStreamsUpdatedListener" ?

Yes , when we have a remoteParticipant , we are listening to the state of the call via addOnVideoStreamsUpdatedListener
